src/lv_core/lv_obj.h

```diff
@@ -200,6 +207,10 @@ typedef struct _lv_obj_t
 
 #if LV_USE_GROUP != 0
     void * group_p; /**< Pointer to the group of the object*/
+#if LV_USE_GROUP_FOCUS_MODE
+    lv_focus_mode_t group_focus_mode;   /**< Group focus mode. */
+    struct _lv_obj_t * group_focus_obj; /**< Pointer to child or parent object. */
```

Comment:
Is it possible for this to reference a nonexistent object (i.e. if the child gets deleted)? I know this is a corner case, but it's better to catch it now than debug it a month later. :slightly_smiling_face:

Reply:
It is enough to do automatic clean-up when one object is deleted? Or there is a need to another validation?

Reply:
Your new fix for this should be fine. Thanks!